active/NavigationView/BottomItems/BottomMenuItems2.md

```diff
@@ -0,0 +1,194 @@
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
+
+Some apps may wish to visually group navigation items into top-aligned and bottom-aligned lists
+(or left- and right-aligned lists), for example:
+
+![NavigationView in Left display mode showing FooterMenuItems](FooterItems_Left.png)
+
+![NavigationView in Top display mode showing FooterMenuItems](FooterItems_Top.png)
+
+While this grouping is possible today by putting NavigationViewItems into the PaneFooter, 
+the approach creates a poor selection and interaction experience as described above.
+That's being corrected in this spec with a new FooterMenuItems property.
+
+As a result, keyboard and assistive technology users will feel that the footer list behavior meets
+their expectations -  the top and bottom menu items will be specified as two separate lists, 
+but it will be smooth to navigate from one to the other. See more in Accessibility section below.
+Selection behavior and animations will also feel better integrated into the whole NavigationView control, 
+as bottom menu items will participate in the same selection model as top menu items. 
+
+# Description
+
+## NavigationView.FooterMenuItems
+
+Use the FooterMenuItems to place navigation items at the end of the navigation pane,
+contrasted with the MenuItems property which places items at the start of the pane.
+
+```cs
+public IList<object> FooterMenuItems { get; }
+```
+
+The default Settings item will be included as the last item in the FooterMenuItems collection. 
+As a result, adding items into FooterMenuItems will by default display them before the Settings item. 
+The Settings item can still be toggled using the `IsSettingsVisible` property. 
+Note: this is the recommended way to remove the Settings item, although it's possible 
+to remove it manually by removing it from the FooterMenuItems collection itself. 
+
+Below is an example of a NavigationView with a FooterMenu where the Mail item is selected.
+
+![NavigationView with FooterMenu](footer-menu-mockup.png)
+
+See also the `FooterMenuItemsSource` property.
+
+## NavigationView.SettingsItem
+
+_This isn't a new property to this spec, but has new behavior remarks here_
+
+Gets the navigation item that represents the entry point to app settings.
+
+```cs
+public object SettingsItem { get; }
+```
+
+### Remarks
+
+This item also appears by default as the last item in the `FooterMenuItems` collection.
+To remove the settings item, though, rather than removing it from the collection,
+use the `IsSettingsVisible` property.
+
+## NavigationView.PaneFooter
+
+_This isn't a new property to this spec, but has new behavior remarks here_
+
+Gets or sets the content for the pane footer.
+
+### Remarks
+
+_In addition to existing remarks ..._
+
+The PaneFooter is placed at the end of the navigation pane, but before the 
+FooterMenuItems. You can place any content into the PaneFooter, but you should not
```

Comment:
I understand that this placement is necessary for compatibility (since it ensures that Settings comes after PaneFooter), but it also prevents a common UI placement of putting non-navigation UI affordances at the end, e.g. a "Sync now" button.

As proposed, the order of items would be

* MenuItems
* PaneFooter ("Sync Now" button goes here)
* FooterMenuItems (includes Settings)

We could fix this by not putting the Settings item in the FooterMenuItems and relying solely on IsSettingsVisible. Then the order would be

* MenuItems
* FooterMenuItems
* PaneFooter
* Settings

You still have the property that PaneFooter comes before Settings, but now you can put "Sync now" in the PaneFooter and it will nestle up against Settings.